etc/spack/defaults/config.yaml

```
@@ -134,6 +134,10 @@ config:
   # enabling locks.
   locks: true
 
+  # The default url fetch method to use. 
+  # If set to 'curl', Spack will require curl on the user's system
+  # If set to 'urllib', Spack will use python built-in libs to fetch
+  url_fetch_method: urllib
```

Should we just call it

```suggestion
  fetch_method: urllib
```

?

---

There are other non-URL fetch methods, such as git/svn/hg.

---

But they also use urls `https://github.com/spack/spack.git` :thinking:

---

Ok, let's keep it `url_fetch_method`, since curl calls itself the tool "for transferring data with URLs" and urllib is called urllib.